oshi-core/src/main/java/oshi/util/ConfigUtil.java

```diff
@@ -0,0 +1,164 @@
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static String get(String key, String def) {
+        return configuration.getProperty(key, def);
+    }
+
+    /**
+     * Get the {@code Integer} property associated with the given key.
+     * 
+     * @param key The property key
+     * @return The property value or {@code 0} if not found
+     */
+    public static int getInteger(String key) {
+        return getInteger(key, 0);
+    }
+
+    /**
+     * Get the {@code Integer} property associated with the given key.
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static int getInteger(String key, int def) {
+        String value = configuration.getProperty(key);
+        return value == null ? def : Integer.parseInt(value);
+    }
+
+    /**
+     * Get the {@code Double} property associated with the given key.
+     * 
+     * @param key The property key
+     * @return The property value or {@code 0.0} if not found
+     */
+    public static Double getDouble(String key) {
+        return getDouble(key, 0.0);
+    }
+
+    /**
+     * Get the {@code Double} property associated with the given key.
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static Double getDouble(String key, double def) {
+        String value = configuration.getProperty(key);
+        return value == null ? def : Double.parseDouble(value);
+    }
+
+    /**
+     * Get the {@code Boolean} property associated with the given key.
+     * 
+     * @param key The property key
+     * @return The property value or {@code false} if not found
+     */
+    public static boolean getBoolean(String key) {
+        return getBoolean(key, false);
+    }
+
+    /**
+     * Get the {@code Boolean} property associated with the given key.
+     * 
+     * @param key The property key
+     * @param def The default value
+     * @return The property value or the default if not found
+     */
+    public static boolean getBoolean(String key, boolean def) {
+        String value = configuration.getProperty(key);
+        return value == null ? def : Boolean.parseBoolean(value);
+    }
+
+    /**
+     * Set the given property, overwriting any existing value.
+     * 
+     * @param key The property key
+     * @param val The new value
+     */
+    public static void set(String key, Object val) {
+        configuration.setProperty(key, val.toString());
+    }
+
+    /**
+     * Load the given {@link Properties} into the global configuration.
+     * 
+     * @param properties The new properties
+     */
+    public static void load(Properties properties) {
+        configuration.putAll(properties);
```

Comment:
I see a way to add more properties but not to `remove()` them.  Or `removeAll()`...